### PR TITLE
Skipping flaky tests

### PR DESCRIPTION
**Pull request by hbandura**
This PR comments/skips tests, builds and benchmarks that are known to be flaky.

Unit tests:
- TestAnnounceGossipQueryMsg #1849
- TestEthClient/TestCallContract #1850
- TestGethClient/TestAccessList #1851

Benchmarks:
- BenchmarkNet1000Txs #1852

Circleci builds:
- end-to-end-cip35-eth-compatibility-test #1853

While creating this PR, the istanbul-e2e-coverage failed once, then worked when it was re-run. I'm not skipping this one right now since I'd like to test it a bit more. Most likely it is because the parent commit has flaky tests, which are run for coverage report.

codecov/project is failing because we are removing a few tests. It is expected.

### CODE REVIEW

**Comment by piersy** [comment]
Coverage from tests in `./e2e_test/...` for `./consensus/istanbul/...` at commit 965b4bf0a1fa4fbea47c53056f9d6a0fa6da2be4
<details><summary>coverage:  47.5% of statements across all listed packages</summary><br>
<pre>
coverage:  57.6% of statements in consensus/istanbul
coverage:  23.7% of statements in consensus/istanbul/announce
coverage:  54.2% of statements in consensus/istanbul/backend
coverage:   0.0% of statements in consensus/istanbul/backend/backendtest
coverage:  24.3% of statements in consensus/istanbul/backend/internal/replica
coverage:  70.6% of statements in consensus/istanbul/core
coverage:  45.0% of statements in consensus/istanbul/db
coverage:   0.0% of statements in consensus/istanbul/proxy
coverage:  75.3% of statements in consensus/istanbul/uptime
coverage: 100.0% of statements in consensus/istanbul/uptime/store
coverage:  51.8% of statements in consensus/istanbul/validator
coverage:  79.2% of statements in consensus/istanbul/validator/random
</pre>
CommentID: 5dd1388110
</details>

File: .circleci/config.yml [hunk 1]
```diff
@@ -545,10 +545,11 @@ workflows:
           requires:
             - checkout-monorepo
             - build-geth
-      - end-to-end-cip35-eth-compatibility-test:
-          requires:
-            - checkout-monorepo
-            - build-geth
+# Flaky!
+#      - end-to-end-cip35-eth-compatibility-test:
```
Comment (YoduYodu) [inline .circleci/config.yml:549]: I think this one is always failing since we merged 1.9.25 and 1.10.7, so probably fix it rather than flakey it

Comment (hbandura) [reply .circleci/config.yml:549]: Probably, yes, I'll open a new issue for it. I didn't want to go into a deeper investigation for each one, rather I'll pass down to the following issues

**Comment by piersy** [comment]
@hbandura Looks good, can you make the associated tickets before we merge this?

**Review by gastonponti — APPROVED** [review]
LGTM

**Pull request merged**